recipes-overlayed/kselftests/kselftests_4.12-rc7.bb

```diff
@@ -3,25 +3,27 @@ LICENSE = "GPLv2"
 LIC_FILES_CHKSUM = "file://COPYING;md5=d7810fab7487fb0aad327b76f1be7cd7"
 
 SRC_URI = "\
-    https://www.kernel.org/pub/linux/kernel/v4.x/linux-${PV}.tar.xz \
+    https://git.kernel.org/torvalds/t/linux-${PV}.tar.gz \
     file://0001-selftests-lib-allow-to-override-CC-in-the-top-level-Makefile.patch \
     file://0001-selftests-timers-use-LDLIBS-to-link-against-libpthread.patch \
-    file://0001-selftests-sigaltstack-fix-packaging.patch \
     file://0001-selftests-seccomp-use-LDLIBS-to-link-against-libpthread.patch \
     file://0001-selftests-gpio-use-pkg-config.patch \
     file://0001-selftests-net-use-LDLIBS-to-link-against-libnuma.patch \
-    file://0001-selftests-breakpoints-allow-to-cross-compile-for-aar.patch;apply=no \
     file://0001-selftests-sync-Skip-the-test-if-kernel-support-is-no.patch \
+    file://fix-building-of-gpio.patch \
```

Comment:
shouldn't we use upstream submitted patch instead? and if not, why should we use yours instead? Because if you disagree with the upstream submitted patch, you should comment on the ml and argue why you think your patch is better.